rules/mandatory-object-comma.js

```diff
@@ -0,0 +1,117 @@
+const { Rule } = require('../api');
+
+module.exports = class extends Rule {
+    getName() {
+        return 'mandatory-object-comma';
+    }
+
+    test(data) {
+        const { formatMessage, addError } = this.outcome;
+
+        this.filterModulesData(data, 'twig').forEach(twigFileData => {
+            const {files, type, name} = twigFileData;
+            const {twig} = files;
+            const {content, path} = twig;
+            const betweenBlocksReg = /^\}[\s]{0,}\{$/i;
+            const twigVariableReg = /\{\{[a-zA-Z0-9 _\=\.\"\%\':\/\_\-\~\,\(\)\?\|\[\]\n\r]{1,}\}\}/gi;
+            const twigVariables = content.match(twigVariableReg);
+            const clearArrayFromBlocksData = stringsArray => stringsArray.filter(
+                string => !string.includes('{%') && !string.includes('%}') && !betweenBlocksReg.test(string));
+
+            const createStringsBetweenBracketsArray = string => {
+                const stringsBetweenBrackets = [];
+                const findStringBetweenBrackets = (string, stringsBetweenBrackets) => {
+                    const reg = /(\[|\]|\{|\})[a-zA-Z0-9 _\=\.\"\%\':\|\-\~\,\(\)\?\n\r]{1,}(\[|\]|\{|\})/i;
+                    const result = string.match(reg);
+
+                    if (!Array.isArray(result)) {
+                        return;
+                    }
+
+                    const stringBetweenBrackets = result[0];
+                    const symbolsCount = stringBetweenBrackets.length;
+                    const index = string.indexOf(stringBetweenBrackets);
+                    const stringAfterBracketsPair = string.slice(index + symbolsCount - 1);
+
+                    stringsBetweenBrackets.push(stringBetweenBrackets);
```

Comment:
Please rename stringsBetweenBrackets for example add list word, because difficult to read stringsBetweenBrackets and stringBetweenBrackets.

Reply:
I renamed to stringsBetweenBracketsList